### PR TITLE
DOC add docstring of advi_minibatch().

### PR DESCRIPTION
**Pull request by taku-y**
I added docstring of advi_minibatch() and removed unused input argument.

### CODE REVIEW

File: pymc3/variational/advi_minibatch.py [hunk 1]
```diff
@@ -193,51 +193,101 @@ def logp_(z_g):
 def advi_minibatch(vars=None, start=None, model=None, n=5000, n_mcsamples=1,
                    minibatch_RVs=None, minibatch_tensors=None, minibatches=None,
                    local_RVs=None, observed_RVs=None, encoder_params=[],
-                   total_size=None, scales=None, optimizer=None, learning_rate=.001,
+                   total_size=None, optimizer=None, learning_rate=.001,
                    epsilon=.1, random_seed=None, verbose=1):
-    """Run mini-batch ADVI.
+    """Perfoem mini-batch ADVI.
```
Comment (twiecki) [inline pymc3/variational/advi_minibatch.py]: Perform

File: pymc3/variational/advi_minibatch.py [hunk 2]
```diff
@@ -193,51 +193,101 @@ def logp_(z_g):
...
 
     Parameters
     ----------
     vars : object
-        Random variables.
+        List of random variables. If None, variational posteriors (normal 
+        distribution) are fit for all RVs in the given model. 
     start : Dict or None
-        Initial values of parameters (variational means).
+        Initial values of parameters (variational means).
```
Comment (twiecki) [inline pymc3/variational/advi_minibatch.py]: Does your editor add these white-spaces?

Comment (taku-y) [reply pymc3/variational/advi_minibatch.py]: I copied the docstring from another file. I use sublime text3.

**Comment by twiecki** [comment]
:+1:

**Pull request merged**